### DISCUSSION TITLE
Why Tags doesn't work?

### DISCUSSION THREAD

**Discussion by roland-varga** [nunocoracao/blowfish#69] [Q&A] [answered]
In my articles tags doesn't show up, why? In other themes, its works...

**Comment by nunocoracao** [comment]
Hello @vargaRolandGit, let's see if I can help. Would you please share some details about the problem you're experiencing with tags? As you can see from the [docs page](https://nunocoracao.github.io/blowfish/) tags are working.

**Reply by nunocoracao** [reply]
@vargaRolandGit moved into discussions as I don't think this is a bug for now

**Reply by roland-varga** [reply]
Okey! So my article works, the thumbnail works too, title & summary good too, but tags, and description does not works. I just installed the theme, ain't changed nothing

**Reply by nunocoracao** [reply]
What do you mean by tags don't work?

1. Can't see tags section on website - similar to https://nunocoracao.github.io/blowfish/tags/
2. Can't see tags below the Article's title in lists? 

In the first case I would need to understand how you are configuring menus and whatnot, is there a repo you can share? If it's the second please configure `article.showTaxonomies`, read [this](https://nunocoracao.github.io/blowfish/docs/configuration/#theme-parameters) for more details.

**Reply by nunocoracao** [reply]
@vargaRolandGit Any updates?

**Reply by R0gue-one** [reply]
@nunocoracao I was having the same problem this helped thanks !

**Comment by nunocoracao** [comment]
Marked this conversation as invalid due to inactivity. @vargaRolandGit let me know if you have new information

**Reply by priyakdey** [reply]
Hi @nunocoracao , I am facing the same issue with Tags and Categories.

The /tags and /categories work, but does not show up on the article of the article list.


`index.md`
```md
---
title: "Clean Code: Factory with Strategy Pattern"
description: "The GoF design patterns helps write more robust, loose cupled and high cohesive applications. This article I am going to talk about a real time scenario where I had to use Factory and Strategy Pattern to solve a particular use case."
summary: "The GoF design patterns helps write more robust, loose cupled and high cohesive applications. This article I am going to talk about a real time scenario where I had to use Factory and Strategy Pattern to solve a particular use case."
#externalUrl: ""
categories: ["software engineering"]
tags: ["design pattern", "factory design pattern", "strategy design pattern", "clean code", "java", "springboot"]
showSummary: true
date: 2021-09-09
draft: true
---

This is an article. !!!

<img src="feature_priyakdey.png" />

## Hello

hello

## okay

These works but not the tags and categories
```
![image](https://user-images.githubusercontent.com/35918976/196043719-a391f7e4-fe1c-4b2b-a16d-ca4537380fb4.png)

![image](https://user-images.githubusercontent.com/35918976/196043768-8b42e3fb-a55a-4373-adca-d9719f84081f.png)

Please let me know if you need any details from the configs

**Reply by nunocoracao** [reply]
Hey @priyakdey open a PR with a fix
https://github.com/priyakdey/priyakdey.github.io/pull/6

Some configs were missing in the params.toml (you could also set the behaviour per article using front-matter I used the global version)

**Comment by nunocoracao** [comment, marked as answer]
Please look at configuration docs
https://nunocoracao.github.io/blowfish/docs/configuration/#theme-parameters

and front-matter docs
https://nunocoracao.github.io/blowfish/docs/front-matter/

In order to understand how to enable the taxonomy badges
